manifests/rules/base.pp

```diff
@@ -72,6 +72,14 @@
     }
   }
 
+# Drop addresses defined in RFC 1122 - Section: 3.2.1.3(g).
```

Comment:
This looks spot on, but could you please add a `Boolean` so that users have an easy way to turn it off? (Enable it by default).

Reply:
Added.